web/src/app/modules/views/side/modules/links-actions/components/links-actions.component.ts

```diff
@@ -16,8 +16,8 @@ export class LinksActions {
 
     public isCollapsed = false;
 
-    public _requirement: Requirement;
-    public _model: IContainer;
+   // public _requirement: Requirement;
+   // public _model: IContainer;
     public _contents: IContainer[];

```

Comment:
Better to delete than to comment.
Also, it seems that _contents and _testSpecifications are not used anywhere. Why not remove them too?